cmd/ebrelayer/relayer/network.go

```diff
@@ -30,15 +30,12 @@ func SetupWebsocketEthClient(ethURL string) (*ethclient.Client, error) {
 	}
 
 	if !IsWebsocketURL(ethURL) {
-		return nil, fmt.Errorf(
-			"invalid websocket eth client URL: %v",
-			ethURL,
-		)
+		return nil, errors.New("invalid websocket eth client URL")
 	}
 
 	client, err := ethclient.Dial(ethURL)
 	if err != nil {
-		return nil, fmt.Errorf("error dialing websocket client: %v", err)
+		return nil, errors.New("error dialing websocket client")
```

Comment:
we can wrap the error here 
```suggestion
		return nil, errors.Wrap(err, "error dialing websocket client")
```

Reply:
Is the `errors.Wrap` syntax compatible with go 1.13? This [go 1.13-errors](https://blog.golang.org/go1.13-errors) article on the golang.org blog recommends wrapping errors with %w.

Reply:
Wrapped with %w in https://github.com/cosmos/peggy/commit/52fdedf6f0b4e4b1c7cf822ab3519dabba3afa17